jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java

```diff
@@ -246,6 +246,32 @@ public void revisitDI(Definitions def) {
         revisitEdgeBoundsInContainers(def);
     }
 
+    public Bounds getShapeBoundsForElement(BaseElement fn, BPMNPlane plane) {
```

Comment:
Please do not add this method. Such method already exists on master branch:

https://github.com/droolsjbpm/jbpm-designer/blob/master/jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java#L583

Feel free to rename the existing method above if you feel that it doesn't describe itself enough.